lib/dal/src/diagram.rs

```diff
@@ -467,7 +468,6 @@ pub struct Diagram {
 
 pub struct DiagramComponentViews {
     pub component_views: Vec<DiagramComponentView>,
-    pub diagram_edges: Vec<SummaryDiagramEdge>,
```

Comment:
This is not a serializable type, so I removed it entirely (only preserving empty edge lists that are serialized right now)